docs/userguide.md

```diff
@@ -299,9 +302,11 @@ kubectl delete federatednamespaceplacement test-namespace
 
 ### Deployment Cleanup
 
-Run the following command to perform a cleanup of the cluster registry and
-federation deployments:
+Assume you are runnig the deloyment cleanup in cluster1. Run the following command to perform
+a cleanup of the cluster registry and federation deployments:
 
 ```bash
 ./scripts/delete-federation.sh cluster2
 ```
+
+The above command will delete both cluster1 and cluster2 from the federation control panel.
```

Comment:
I would replace "delete" with "unjoin" just to match the `kubefed2` command or provide one in parenthesis e.g. delete (unjoin).

Reply:
done